Library/Homebrew/formula.rb

```diff
@@ -1626,6 +1626,12 @@ def opt_or_installed_prefix_keg
     end
   end
 
+  alias any_installed_prefix opt_or_installed_prefix_keg
```

Comment:
Can we have a new function with a (commented out) `odeprecated` here? That'll allow us to deprecate it for 2.5.0 rather than keeping the alias around forever.